crates/pop-parachains/src/utils/helpers.rs

```diff
@@ -24,6 +24,35 @@ pub(crate) fn sanitize(target: &Path) -> Result<(), Error> {
 	Ok(())
 }
 
+pub fn is_initial_endowment_valid(initial_endowment: &str) -> bool {
+	match initial_endowment.parse::<u128>() {
+		Ok(_) => return true,
+		Err(_error) => match left_shift(initial_endowment) {
+			Ok(_) => return true,
+			Err(_error) => return false,
+		},
+	}
+}
+fn left_shift(initial_endowment: &str) -> Result<u64, Error> {
+	let v: Vec<&str> = initial_endowment.split(" << ").collect();
+	if v.len() < 2 {
+		return Err(Error::EndowmentError);
+	}
+	let left = v[0]
+		.split("u") // parse 1u64 characters
+		.take(1)
+		.collect::<String>()
+		.parse::<u64>()
+		.or_else(|_e| Err(Error::EndowmentError))?;
+	let right = v[1]
+		.chars()
+		.filter(|c| c.is_numeric()) // parse 1u64 characters
+		.collect::<String>()
+		.parse::<u32>()
```

Comment:
Can this not be u8? 

Can't shift more than 127 bits for a u128

Reply:
`checked_shl` for u128 is expecting a u32: https://doc.rust-lang.org/std/primitive.u128.html#method.checked_shl